pennylane/math/single_dispatch.py

```diff
@@ -47,6 +47,8 @@ def _i(name):
 # qml.SparseHamiltonian are not automatically 'unwrapped' to dense NumPy arrays.
 ar.register_function("scipy", "to_numpy", lambda x: x)
 ar.register_function("scipy", "shape", np.shape)
+ar.register_function("scipy", "conj", np.conj)
+ar.register_function("scipy", "transpose", np.transpose)

```

Comment:
This allows us to take the conjugate-transpose of a scipy sparse matrix without deprecation warnings.